crates/turborepo-repository/src/package_manager/bun.rs

```diff
@@ -27,10 +28,12 @@ impl Iterator for BunDetector<'_> {
         }
 
         self.found = true;
-        let package_json = self.repo_root.join_component(LOCKFILE);
+        let bun_lock = self.repo_root.join_component(LOCKFILE);
 
-        if package_json.exists() {
+        if bun_lock.exists() {
             Some(Ok(PackageManager::Bun))
+        } else if self.repo_root.join_component(LOCKFILE_BINARY).exists() {
+            Some(Err(Error::BunBinaryLockfile))
```

Comment:
did these lines introduce a regression regarding binary bun lockfiles or were they not supported to begin with?

<img width="642" alt="image" src="https://github.com/user-attachments/assets/96e47c17-38be-482c-ab73-be9261bb0618" />

Reply:
just wondering if i should be opening a new issue since moving to the text based lockfile is not an option currently, as the issue with the empty root package name key still exists.

https://github.com/vercel/turborepo/issues/9628#issuecomment-2662589506